desktop/contextBridge.js

```diff
@@ -0,0 +1,40 @@
+const _ = require('underscore');
+const {
+    contextBridge,
+    ipcRenderer,
+} = require('electron');
+const ELECTRON_EVENTS = require('./ELECTRON_EVENTS');
+
+const WHITELIST_CHANNELS_RENDERER_TO_MAIN = [
+    ELECTRON_EVENTS.REQUEST_FOCUS_APP,
+    ELECTRON_EVENTS.REQUEST_UPDATE_BADGE_COUNT,
+    ELECTRON_EVENTS.REQUEST_VISIBILITY,
+    ELECTRON_EVENTS.START_UPDATE,
+];
+
+const WHITELIST_CHANNELS_MAIN_TO_RENDERER = [
+    ELECTRON_EVENTS.SHOW_KEYBOARD_SHORTCUTS_MODAL,
+    ELECTRON_EVENTS.UPDATE_DOWNLOADED,
+];
+
+contextBridge.exposeInMainWorld('electron', {
+    send: (channel, data) => {
+        if (!_.contains(WHITELIST_CHANNELS_RENDERER_TO_MAIN, channel)) {
+            // eslint-disable-next-line no-console
+            console.warn(`Attempt to send data across electron context bridge with invalid channel ${channel}`);
```

Comment:
Should I try to figure out how to send a `warning` log to the server (on staging/production only)? 🤔

Reply:
Any context on what would be calling `send()` with a channel that is not whitelisted or how it could happen? And if it did, would we need to know about it on the server?

Reply:
I tried for a while to think about an attack vector here and I couldn't come up with one (because even if I could send a non-whitelisted event to `main`, it wouldn't do anything if the main process wasn't listening for that event? I'm not an expert though so I'm not 100% confident.

That said, I guess we can just leave this console warning here (it might be useful for development in the case you're trying to add a new event and forget to add it to the whitelist.) It would be pretty involved to set up server logging right here anyways.

Reply:
>it might be useful for development in the case you're trying to add a new event and forget to add it to the whitelist

Would it be even more useful to just throw an `Error` here? If the log is useful on dev then I'd probably notice an Error faster than a `console.warn()`.

Reply:
Great suggestion, implemented 👍